runhouse/main.py

```diff
@@ -666,6 +341,96 @@ def cluster_list(
         print_sky_clusters_msg(len(sky_clusters))
 
 
+@cluster_app.command("keep_warm")
+def keep_warm(
+    cluster_name: str,
+    up: Optional[bool] = typer.Option(False, help="Start the cluster"),
```

Comment:
maybe this should be a separate command, since keep warm really implies that the cluster is already up. wdyt?